Index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+  <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js" integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV" crossorigin="anonymous"></script>
+</head>
+<body>
+  <h1>Hello World, from Ellah and Alex team</h1>
+
+  <p id="main"> 
+   <label id="title"> My Beautiful Heritage  </label>   
+  
+   <div id="img-div">
+      <img src="https://i.ytimg.com/vi/zFXEkjY4dlk/maxresdefault.jpg" alt="some image" class="responsive center" >
+     <br> <label id="img-caption">Refreshing Flower as Microsoft</label>
+  
+  
+   </div>
+  
+   <div id="Tribute-info"   class="card text-white bg-primary mb-3">
+      <div class="card-header">Header</div>
+      <div class="card-body">
+        <h4 class="card-title">Primary card title</h4>
+        <p class="card-text">My heritage is my tradition.  I love gardening and i love flowers they make the world a beautiful place. </p>
+      </div>
+    </div>
+    <a id="Tribute-Link" rel="noopener noreferrer" target="_blank" href="https://github.com/elena-in-code/Tribute-page">Come share our heritage here!</a>
```

Comment:
I think it would be better to bring some style to the Hyperlinks, to have a great look when you use it.

Reply:
:+1: Great Comment.